cli/view.js

```diff
@@ -82,7 +82,7 @@ const getAuspiceBuild = () => {
     cwd !== sourceDir &&
     fs.existsSync(path.join(cwd, "index.html")) &&
     fs.existsSync(path.join(cwd, "dist")) &&
-    fs.readdirSync(path.join(cwd, "dist")).filter((fn) => fn.match(/^auspice.bundle.[a-z0-9]+.js$/)).length === 1
+    fs.readdirSync(path.join(cwd, "dist")).filter((fn) => fn.match(/^auspice.main.bundle.[a-z0-9]+.js$/)).length === 1
```

Comment:
I noticed in passing that those `.` should be escaped in the regex since they're meant to be literals.

Reply:
Good catch, thanks!

Reply:
@victorlin - I'm going to fix this now (it was originally my oversight) as this'll allow multiple PRs to be merged

Reply:
Actually scrap that, let's fix this over in https://github.com/nextstrain/auspice/pull/1611